src/mmw/apps/core/decorators.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import rollbar
+
+from django.utils.timezone import now
+from django.conf import settings
+
+from apps.core.models import (Job,
+                              RequestLog)
+
+
+rollbar_settings = getattr(settings, 'ROLLBAR', {})
+if rollbar_settings:
+    rollbar.init(rollbar_settings.get('access_token'),
+                 rollbar_settings.get('environment'))
+
+
+def log_request(view):
+    """
+    Log the request and its response as a RequestLog model
+    """
+
+    def decorator(request, *args, **kwargs):
+        requested_at = now()
+
+        view_result = view(request, *args, **kwargs)
+
+        user = request.user if request.user.is_authenticated() else None
+
+        response_time = now() - requested_at
+        response_ms = int(response_time.total_seconds() * 1000)
+
+        log = RequestLog.objects.create(
+            user=user,
+            job=view_result.data.get('job', None)
```

Comment:
Should this be `job_uuid` instead of `job`?

Reply:
Yes